app/pages/DashboardV2.tsx

```diff
@@ -0,0 +1,211 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
Is this still necessary? Looks like a duplicate of Dashboard

Reply:
This was created as a point to build off from the old dashboard. We can remove the old one once the new dashboard is set then keep adding the abstracted provider imports to the new one.